tests/functional/u/used/used_before_assignment.py

```diff
@@ -13,3 +13,8 @@ def inner():
         pass
 
 outer()
+
+import time  # pylint: disable=unused-import, wrong-import-position
+def redefine_time_import():
+    print(time.time())  # [used-before-assignment]
```

Comment:
Hmm, I learned something about python thanks to this. I would not have expected it to crash. Well I suppose it's a good thing that I never try to do this kind of thing 😄